### ISSUE TITLE
Online Subtitle Search feature should allow user to enter custom search string

### ISSUE THREAD

**Issue by candh** [iina/iina#2569]
**System and IINA version:**

- macOS 10.14.2 
- IINA 1.0.4 Build 107 

**Expected behavior:**
Online Subtitle Search feature should allow user to enter custom search string. 

**Rationale:**
When IINA is used as streaming client, often the URL will be like: "http://localhost:6969". `FileHandle(forReadingFrom: url)` would then fail when supplied this URL and would not acquire a Handle. The error is then shown: "Error reading file" on the screen.

It would be nice if users could manually enter search text, in this case, or otherwise by pressing a button in the subtitle sidebar. User would then be showed a list of available subtitles to choose from and then users can pick one (which is already shown if everything goes well). MX Player on Android provides this functionality

**Comment by maharjanmilan** [comment]
I totally agree with @candh When I stream from url, often the name of the file is not correct and IINA cannot search for the subtitle. A feature to insert a custom name would be very very helpful.
